exec/vector/src/main/codegen/templates/VariableLengthVectors.java

```diff
@@ -897,7 +901,7 @@ public BufferedMutator(int startIdx, int buffSz, ${minor.class}Vector parent) {
       this.parent = parent;
       this.dataBuffOff = this.parent.offsetVector.getAccessor().get(startIdx);
       this.totalDataLen = this.dataBuffOff;
-      this.offsetsMutator = new UInt4Vector.BufferedMutator(startIdx, buffSz * 4, parent.offsetVector);
+      this.offsetsMutator = new UInt4Vector.BufferedMutator(startIdx, buffSz, parent.offsetVector);
```

Comment:
Why remove ` * 4 `?

Reply:
Performance benchmarks indicated that 4kb is an optimal value; larger values will have the side effect of overflowing from the L1 cache (32kb). Actually, I meant to do this change a while back but somehow missed this one.